src/mlpack/methods/ann/layer/dropconnect.hpp

```diff
@@ -164,7 +164,7 @@ class DropConnect
   double scale;
 
   //! Locally-stored weight object.
-  OutputDataType parameters;
+  OutputDataType weights;
```

Comment:
Did you mean to also implement `WeightSize()` for `DropConnect`?

Reply:
Yes, but I could not figure out how to. There are two constructors but one does nothing and the other one takes `inSize` and `outSize` but passes it over to the linear layer. Not sure what `WeightSize` should return, maybe 0?

Reply:
`DropConnect`  has to weights, so in each case, we should return 0.

Reply:
I see, actually, it had both `Model()` and `Parameters()` function, so I thought we had to include the size of other layers in the model as well.

Reply:
It looks good to me now.  Thanks! :+1: